addons/humanizer/scripts/resources/mhclo.gd

```diff
@@ -84,11 +84,12 @@ func parse_file(filename:String):
 						line_dict.weight = []
 						line_dict.offset = Vector3.ZERO	
 						for i in 3:
-							line_dict.vertex.append(line_array[i])
-							line_dict.weight.append(line_array[i+3])	
-						line_dict.offset.x = line_array[6]		
-						line_dict.offset.y = line_array[7]	
-						line_dict.offset.z = line_array[8]		
+							line_dict.vertex.append(int(line_array[i]))
```

Comment:
the data before in the debugger was all floats, this explicitly types the data so you always pass and create the correct data in the array, a vertex_id cannot be a float.